Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhinc/audit/transform/AuditTransformsTest.java

```diff
@@ -219,6 +220,17 @@ protected void testGetActiveParticipantDestination(LogEventRequestType request,
             webContextProperties.getProperty(NhincConstants.REMOTE_HOST_ADDRESS));
     }
 
+    protected void testAuditSourceIdentification(List<AuditSourceIdentificationType> auditSourceIdentification, AssertionType assertion) {
+        if (assertion != null && assertion.getUserInfo() != null && assertion.getUserInfo().getOrg() != null
+            && assertion.getUserInfo().getOrg().getHomeCommunityId() != null
+            && auditSourceIdentification != null && !auditSourceIdentification.isEmpty()
+            && auditSourceIdentification.size() > 0) {
+            for (AuditSourceIdentificationType auditSourceId : auditSourceIdentification) {
+                assertEquals(AuditTransformsConstants.HOME_COMMUNITY_ID_PREFIX + assertion.getUserInfo().getOrg().getHomeCommunityId(), auditSourceId.getAuditSourceID());
```

Comment:
Please wrap this line; also check the PR for other lines that are cut off (look for files that have a horizontal scroll bar on GitHub)